rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -27,8 +27,7 @@ module LegacySupport
            send_keys_to_active_element get_window_handles get_current_window_handle
```

Comment:
That seems like it should be a separate piece to me, but I wouldn't argue against it. If you have a game plan, let me know what I can help with.